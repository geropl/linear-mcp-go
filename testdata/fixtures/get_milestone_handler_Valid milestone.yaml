---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 296
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tquery ProjectMilestone($id: String!) {\n\t\t\tprojectMilestone(id: $id) {\n\t\t\t\tid\n\t\t\t\tname\n\t\t\t\tdescription\n\t\t\t\ttargetDate\n\t\t\t\tproject {\n\t\t\t\t\tid\n\t\t\t\t\tname\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"id":"c86acc00-3035-4a67-82f2-2a5bf6453e92"}}'
        form: {}
        headers:
            Content-Type:
                - application/json
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"data":{"projectMilestone":{"id":"c86acc00-3035-4a67-82f2-2a5bf6453e92","name":"Updated Milestone Name","description":"Updated Description","targetDate":"2025-01-01","project":{"id":"bfa49864-16c9-44db-994e-a11ba2b386f1","name":"Updated Project Name 2"}}}}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"102-BS+utz/wRqBQhmWIePACw/zkABE"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 0s
