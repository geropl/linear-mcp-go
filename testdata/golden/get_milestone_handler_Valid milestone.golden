err: ""
output: |
    Milestone: Updated Milestone Name
      ID: c86acc00-3035-4a67-82f2-2a5bf6453e92
      Description: Updated Description
      Target Date: 2025-01-01
      Project: Updated Project Name 2 (bfa49864-16c9-44db-994e-a11ba2b386f1)
